packages/e2e-test-utils/README.md

```diff
@@ -16,7 +16,7 @@ npm install @wordpress/e2e-test-utils --save-dev
 
 ### activatePlugin
 
-[src/index.js#L1-L1](src/index.js#L1-L1)
+[src/index.js#L2-L2](src/index.js#L2-L2)
```

Comment:
@nosolosw - this only confirms that we need to find a way to output a line in the source code where all API methods are defined rather than where they are exported. In the majority of cases, we will use `index.js` file to collect public API methods from files which contain the actual implementation.